### ISSUE TITLE
Error in latest adding/creating folders

### ISSUE THREAD

**Issue by Tupsi** [earlSt1/vtt-compendium-folders#56]
Just pulled the version 2.2.1 and now I get this when I try to put a compendium into a folder.

```
TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Object'
    |     property 'children' -> object with constructor 'Array'
    |     index 0 -> object with constructor 'Object'
    --- property 'parent' closes the circle
    at JSON.stringify (<anonymous>)
    at duplicate (foundry.js:978)
    at CompendiumFolder._getSaveData (compendium-folders.js:332)
    at CompendiumFolder.save (compendium-folders.js:388)
    at CompendiumFolderEditConfig._updateObject (compendium-folders.js:1553)
    at CompendiumFolderEditConfig._onSubmit (foundry.js:5251)

```

and the action does not get completed successfully.
Was working fine with the old version.

Creating a new folder still works, but putting anything in it afterwards fails.

**Comment by earlSt1** [comment]
When you say working fine with old version, is this 2.2.0?

Also what FVTT version + system are you using?

**Comment by Tupsi** [comment]
> When you say working fine with old version, is this 2.2.0?
I can not says exactly, just that I update all modules regularly (read almost every day atm). so 2.2.0 seems likely.
> 
> Also what FVTT version + system are you using?
0.7.9 on a debian linux.

**Comment by earlSt1** [comment]
Sorry i meant game system :smile:

**Comment by Tupsi** [comment]
> Sorry i meant game system 😄

lol, ok. We use DND5e.

**Comment by earlSt1** [comment]
Hmm ok, is it any folder that is affected or a specific folder?

I havent run into this during testing which is strange.... I can see why its complaining though.

**Comment by Tupsi** [comment]
creating a new folder works and adding one compendium during this process works as well. 
If I edit this newly created folder to add a second compendium I get the error.

**Comment by earlSt1** [comment]
Righto. could you try uninstalling and installing using the following testing manifest to see if it fixes it?

https://raw.githubusercontent.com/earlSt1/vtt-compendium-folders/master/test/module.json

**Comment by Tupsi** [comment]
> Righto. could you try uninstalling and installing using the following testing manifest to see if it fixes it?
> 
> https://raw.githubusercontent.com/earlSt1/vtt-compendium-folders/master/test/module.json

works.

**Comment by earlSt1** [comment]
Fixed in v2.2.2
